payment-failure/build.sbt

```diff
@@ -23,6 +23,7 @@ libraryDependencies ++= Seq(
   ("org.jlib" % "jlib-awslambda-logback" % "1.0.0").exclude("org.slf4j", "log4j-over-slf4j"),
```

Comment:
This library seems to only have this one release in 2018.  this brings in `h.qos.logback:logback-classic:1.3.0-alpha4` which is the one that has the vulnerability